docs/usage/shoot_settings/default_seccomp_profile.md

```diff
@@ -0,0 +1,28 @@
+---
+description: Enable the use of `RuntimeDefault` as the default seccomp profile through `spec.kubernetes.kubelet.seccompDefault`
+weight: 15
+---
+
+# Default Seccomp Profile and Configuration 
+
+This is a short guide describing how to enable the defaulting of seccomp profiles for shoot clusters. Running pods in `Unconfined` (seccomp disabled) mode is undesirable since this is the least restrictive profile. Also, mind that any privileged container will always run as `Unconfined`. More information about seccomp can be found in this [Kubernetes tutorial](https://kubernetes.io/docs/tutorials/security/seccomp/).
```

Comment:
```suggestion
This is a short guide describing how to enable the defaulting of seccomp profiles for shoot clusters. Running pods in `Unconfined` (seccomp disabled) mode is undesirable since this is the least restrictive profile. Also, mind that any privileged container will always run as `Unconfined`. More information about seccomp can be found in [Restrict a Container's Syscalls with seccomp](https://kubernetes.io/docs/tutorials/security/seccomp/).
```